hadoop-ozone/recon-codegen/src/main/java/org/hadoop/ozone/recon/schema/ReconTaskSchemaDefinition.java

```diff
@@ -64,6 +64,7 @@ private void createReconTaskStatusTable(Connection conn) {
         .column("task_name", SQLDataType.VARCHAR(766).nullable(false))
         .column("last_updated_timestamp", SQLDataType.BIGINT)
         .column("last_updated_seq_number", SQLDataType.BIGINT)
+        .column("last_task_successful", SQLDataType.BIT)
```

Comment:
this is similar to below column getting added, this can be removed as datatype is also different.
  .column("last_task_run_status", SQLDataType.INTEGER)